app/assets/javascripts/controllers/FileCtrl.coffee

```diff
@@ -1,18 +1,15 @@
-@bruseApp.controller 'FileCtrl', ['FileHandler', '$scope', '$http', (FileHandler, $scope, $http) ->
+@bruseApp.controller 'FileCtrl', ['FileHandler', '$scope', '$rootScope', '$location', '$routeParams', (FileHandler, $scope, $rootScope, $location, $routeParams) ->
   $scope.files = []
   $scope.bruse_files = []
   $scope.new_files = []
   $scope.loading = true
 
-  # get identity information from add view
-  $scope.identity = IDENTITY_PARAMS
```

Comment:
In the branch feature/drive-files the IDENTITY_PARAMS changed to contain service as well. So the change to get it to match develop now is not large. $routeParams.identity_id only has to be changed to $routeParams.identity.id(?)